templates/python/custom-copilot-basic/env/.env.local.user.tpl

```diff
@@ -18,7 +18,7 @@ SECRET_AZURE_OPENAI_API_KEY='{{{azureOpenAIKey}}}'
 {{^azureOpenAIKey}}
 SECRET_AZURE_OPENAI_API_KEY=
 {{/azureOpenAIKey}}
-SECRET_AZURE_OPENAI_MODEL_DEPLOYMENT_NAME=
+AZURE_OPENAI_MODEL_DEPLOYMENT_NAME=
 {{#azureOpenAIEndpoint}}
 SECRET_AZURE_OPENAI_ENDPOINT='{{{azureOpenAIEndpoint}}}'
```

Comment:
Please also remove the secret prefix for this env

Reply:
removed